spec/models/vm_or_template_spec.rb

```diff
@@ -335,6 +335,19 @@
     end
   end
 
+  context "#resource_group" do
+    before do
```

Comment:
On second thought... Do we even need this test?  It feels like we're testing rails here.